client/packages/common/src/ui/layout/tables/material-react-table/components/ChipTableCell.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Box, Chip } from '@openmsupply-client/common';
+import { MRT_Cell, MRT_RowData } from 'material-react-table';
+
+interface ChipTableCellProps<T extends MRT_RowData> {
+  cell: MRT_Cell<T>;
+}
+
+/**
+ * Displays a list of labels in a table cell.
+ *
+ * This expects that the data accessor returns a list of string for a given table row.
+ */
+export const ChipTableCell = <T extends MRT_RowData>({
+  cell,
+}: ChipTableCellProps<T>) => {
+  const items = cell.getValue<string[]>();
+
+  if (!items || items.length === 0) return null;
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexWrap: 'wrap',
+        gap: 0.5,
+        maxWidth: 250,
+        padding: '3px 0',
```

Comment:
Don't need the `padding` or the `maxWidth` -- there's already enough padding from the table cell (and it looks worse in "dense" mode), and with the user being able to customise their column width, we don't want to restrict it.